README.md

```diff
@@ -1,12 +1,12 @@
 ﻿# Co-Authors Plus
 
-Stable tag: 3.5.15  
-Requires at least: 4.1  
-Tested up to: 6.3  
-Requires PHP: 5.6  
-License: GPLv2 or later  
-License URI: https://www.gnu.org/licenses/gpl-2.0.html  
-Tags: authors, users, multiple authors, co-authors, multi-author, publishing  
+Stable tag: 3.6.0
+Requires at least: 4.1
+Tested up to: 6.5
+Requires PHP: 5.6
+License: GPLv2 or later
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
```

Comment:
Looking at [`c216c0d` (#1032)](https://github.com/Automattic/Co-Authors-Plus/pull/1032/commits/c216c0d2d30eb6a03eaa5df9accb39103e527b8a) it seems that the required double-trailing space for Markdown linebreaks have been stripped inadvertently. They should be restored.